composer.json

```diff
@@ -16,12 +16,12 @@
         "php": ">=5.3.3",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "guzzle/guzzle": "^2.8|^3.0",
+        "guzzle/guzzle": "^2.8||^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.4|^3.0",
-        "symfony/console": "^2.1|^3.0",
-        "symfony/stopwatch": "^2.2|^3.0",
-        "symfony/yaml": "^2.1|^3.0"
+        "symfony/config": "^2.4||^3.0",
```

Comment:
Are you extra sure this is good change?
I have never seen this in any other repos...(beside composer one).

* Keep the PR atomic - one change = one PR
* [|](http://semver.mwl.be/#?package=symfony%2Fconfig&version=%5E2.4%20%7C%20%5E3.0&minimum-stability=stable) vs [||](
http://semver.mwl.be/#?package=symfony%2Fconfig&version=%5E2.4%20%7C%7C%20%5E3.0&minimum-stability=stable)